src/ovp_pipeline/commands/_chats_list_page.py

```
@@ -0,0 +1,200 @@
+def _group_by_status(rows: list[ChatRow]) -> dict[str, list[ChatRow]]:
+    grouped: dict[str, list[ChatRow]] = {s: [] for s in _STATUS_ORDER}
+    for row in rows:
+        status = row.status if row.status in grouped else "active"
+        grouped[status].append(row)
+    return grouped
+
+
+def _render_chat_row(row: ChatRow) -> str:
+    anchor_label = row.anchor_title or row.anchor_ref or row.anchor_kind
+    anchor_pill = (
+        f'<span class="pill ghost">{escape(row.anchor_kind)}: ' f"{escape(anchor_label)}</span>"
```

Ruff (0.15.12)

[warning] Implicitly concatenated string literals on one line

(ISC001)